### DISCUSSION TITLE
set units to tariff your not on for testing

### DISCUSSION THREAD

**Discussion by popeter1** [BottlecapDave/HomeAssistant-OctopusEnergy#217] [Ideas]
so currently on Flexible Octopus but looking to switch to Agile at some point but want to make sure my automations are working before making the switch and to test if that would save me money before switching

as the API lets to view unit rates of other tarrifs would be be possible to add a select tariff in configure?

**Comment by BottlecapDave** [comment]
Hello and welcome. I've moved your request to an [issue](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/221) to track the progress, which has been added to the backlog.
